hypermodularization/what-is-hypermodular.md

```diff
@@ -0,0 +1,53 @@
+## What is "hypermodularization"?
+
+What I[^a] mean by this is a project (examples: [React](https://facebook.github.io/react/ 'Facebook Code: React'), [redux](http://redux.js.org/ 'redux')) that is explicitly designed with the expectation that a single dependency will **not** be self-contained. A developer-user (henceforth: user) will have to install several, perhaps even “many”[^b], dependencies in order to make full use of the project/framework or perhaps even to use it at all (as intended).
```

Comment:
Yeah fair enough. I think what I really meant is sort of "self-contained for most use cases" or something like that. It'd be interesting to know what the real numbers are, but I'd guess like 99% of the people out there using `react` in a piece of deployed production software (not tooling) have `react-dom` or `react-native` (or another renderer if there are any other major ones? not sure) as an additional dependency.

Like your edit - will incorporate, thanks!